Sources/PubNubChat/PubNub/Models/ChatMessage.swift

```diff
@@ -56,8 +56,8 @@ public struct ChatMessage<Custom: ChatCustomData>: Identifiable, Hashable, Codab
       id: String = UUID().uuidString,
       text: String = String(),
       contentType: String? = nil,
-      content: JSONCodable? = nil,
-      custom: Custom.Message = Custom.Message(),
+      content: Custom.Message.Content? = nil,
```

Comment:
Can this be made non-optional, but default to Void in the event the customer doesn't want to set this value?  Just want to be consistent with other generic values.